src/main/resources/changesets/changelog-consent-82.0.xml

```diff
@@ -1,6 +1,8 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <!--Create the dar_dataset table-->
```

Comment:
We can't make modifications to this file after it has already been executed on a database. Any schema changes need to be in a separate changelog file.

Reply:
Ooopps sorry forgot to remove the comment